src/metabase/search/task/search_index.clj

```diff
@@ -127,16 +96,7 @@
     (task/schedule-task! job trigger)))
 
 (defmethod task/init! ::SearchIndexUpdate [_]
-  (let [job         (jobs/build
-                     (jobs/of-type SearchIndexUpdate)
-                     (jobs/with-identity update-job-key))
-        trigger-key (triggers/key (str update-stem ".trigger"))
-        trigger     (triggers/build
-                     (triggers/with-identity trigger-key)
-                     (triggers/for-job update-job-key)
-                     (triggers/start-now))]
-    (task/schedule-task! job trigger)))
+  (ingestion/start-listener!))
```

Comment:
Interesting idea to keep using this hook! Two concern though:

1. The `task/init!` method is documented as specifically pertaining to Quartz jobs.
2. We don't get coordination on shutting down the listeners when the server stops.

Not blocking for me, as (1) is cosmetic and will shake out eventually and (2) isn't a regression. Still suggest going the way of a "listener registry" though!